slang-antlr/src/test/java/com/sonarsource/slang/antlr/SLangConverterTest.java

```diff
@@ -472,6 +474,79 @@ public void stringLiterals() {
     }
   }
 
+  @Test
+  public void jump() {
+    String slangCode = "break foo";
+    Tree tree = converter.parse(slangCode + ";");
+
+    assertTree(tree).isNotNull();
+    assertTree(tree).isInstanceOf(TopLevelTree.class);
+    TopLevelTree topLevelTree = (TopLevelTree) tree;
+    assertThat(topLevelTree.declarations()).hasSize(1);
+
+    JumpTree jumpTree = (JumpTree) topLevelTree.declarations().get(0);
+    assertTree(jumpTree).isNotNull();
+    assertTree(jumpTree).isInstanceOf(JumpTree.class);
```

Comment:
Probably this assertion and one above are useless as you cast before. Casting will fail if class is not `JumpTree` or value is `null`